{{ cookiecutter.package_name }}/pyproject.toml

```
@@ -40,9 +40,10 @@ tests = [
   "pytest-xdist",
 ]
 docs = [
-  "sphinx",
-  "sphinx-automodapi",
-  "sunpy-sphinx-theme",
+  "sphinx>=5.0.0",
+  "sphinx-automodapi>=0.14.1",
+  "sphinx-changelog>=1.5.0",
+  "sunpy-sphinx-theme>=2.0.3",
```

To minimise conflicts:
```suggestion
  "sphinx",
  "sphinx-automodapi",
  "sunpy-sphinx-theme",
  "sphinx-changelog",
```

---

but that order is quite upsetting, but I do understand

---

yeah it's not great, we could put the theme last, it will probably conflict everywhere anyway lol